src/main/java/core/mate/academy/model/Machine.java

```diff
@@ -1,8 +1,17 @@
 package core.mate.academy.model;
 
 public abstract class Machine implements Workable {
-    private String name;
     private String color;
+    private String name;
+
+    public Machine() {
+
+    }
+
+    public Machine(String name, String color) {
```

Comment:
make it protected, public constructor for abstract class doesn't really work, only subclasses can call this constructor anyway